client/app/lib/views/credentiallist/accountcredentiallistcontroller.coffee

```diff
@@ -301,6 +301,7 @@ module.exports = class AccountCredentialListController extends KodingListControl
       then @addItem(credential).verifyCredential()
       else
         @addItem credential
+        @loadItems()
```

Comment:
No need to call super class after adding a new credential.
Better to give index as a second argument to addItem function.
https://github.com/koding/kd/blob/master/lib/components/list/listviewcontroller.coffee#L168
`@addItem credential, 0` would add item to top
/cc: @sinan